City-of-Helsinki/helfi-kymp-district-project-search
# UHF-6218: Search result list styling

# [UHF-6218](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6218)


## What was done


* Add listing with correct style and data from index


## How to install and test

Testing with Helfi KYMP instance:
* You need helfi-kymp drupal locally with UHF-6253_add_elastic branch.
* Check that there are Districts and Projects added with correct data
* Remember to update the elastic indexes: https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/admin/config/search/search-api/index/projects, https://helfi-kymp.docker.so/fi/kaupunkiymparisto-ja-liikenne/admin/config/search/search-api/index/districts
* Add District and Project search paragraph into a page.
* You should see the listing on the page if javascript is not disabled.

Testing with React App front
* Make sure your instance is up and running on correct branch.
  * `git pull origin UHF-6218_search_result_list_styling`
  * `npm i`
* Run `npm start`
* You should see the listing

## How to test


* [x] Check that list look similar to designs: https://app.abstract.com/projects/163e84f5-4ae6-4649-95b0-638f88896d87/branches/58b35439-47a9-47ae-9e94-177eb545dc8d/commits/5189c882521352b79cbc2be55cae038c319d78d7/files/191bca1c-2f2c-4767-b5cc-cd8635d28c52/layers/A5B93F2A-9ECE-4ED0-BB62-E9D787267C94?collectionId=2a9e26e4-fe41-4e87-8d2a-0af829c949d7&collectionLayerId=5831eaa4-d5bf-4439-a129-e5cbb0f145ff&mode=build

## Designers review



## Other PRs


* Drupal Helfi KYMP repo: https://github.com/City-of-Helsinki/drupal-helfi-kymp
